### DISCUSSION TITLE
getting an error after creating project

### DISCUSSION THREAD

**Discussion by Muhammad-zain-rehman** [vercel/next.js#50839] [Help]
### Summary

I'm new at NEXT js road, I just following the Next js documentation by following the same steps as per documentation I'm getting an error

my node version: is v14.17.0 (the same version is being used by other projects as well so i can't update my node version)

step 01: I followed the both **npx create-next-app@latest**  and also **npx create-next-app**
step 02: after installation dependencies, I run **(npm run dev)**

![image](https://github.com/vercel/next.js/assets/85789232/0c8d7c25-95cc-466e-acc7-f074171afa67)
![image](https://github.com/vercel/next.js/assets/85789232/48badbcc-6739-4766-a4dc-d85e8dacbbc3)

**Comment by ciruz** [comment]
Hey, the latest next.js requires Node.js 16.8 or later, see: https://nextjs.org/docs/getting-started/installation

PS: you can make use of [nvm](https://github.com/nvm-sh/nvm) to switch between node versions simple with commands like:
```bash
$ nvm use 16
Now using node v16.9.1 (npm v7.21.1)

$ nvm use 14
Now using node v14.18.0 (npm v6.14.15)
```

So it will be easier for you, to work with different Node.js versions
